db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java

```diff
@@ -1942,6 +1942,8 @@ The message identifier (ie. the value that matches a field in this class)
 	String LANG_CANCELLATION_EXCEPTION							   = "SE008";
 	String LANG_INVALID_DAY										   = "SE022";
 	String ERROR_PARSING_EXCEPTION								   = "SE023";
+	String PARAMETER_CANNOT_BE_NULL								   = "SE024";
```

Comment:
This is redundant with

String PARAMETER_CANNOT_BE_NULL									= "TS006";

Reply:
I thought the "TS" prefix was specific to the region operations, that's why I moved it into the SE (Splice Engine) exception codes.